runtime/cere-dev/src/lib.rs

```diff
@@ -430,6 +441,10 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = frame_system::Pallet<Runtime>;
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
+	type FreezeIdentifier = ();
```

Comment:
We probably should not use the default value of `()` in production / dev runtimes